packages/webcomponents/src/components/business-forms/business-form/business-form.tsx

```diff
@@ -29,6 +30,24 @@ export class BusinessForm {
   @Event({ eventName: 'click-event' }) clickEventNew: EventEmitter<BusinessFormClickEvent>;
   @Event({ eventName: 'clickEvent' }) clickEventOld: EventEmitter<BusinessFormClickEvent>;
 
```

Comment:
In `business-form` - I am correctly seeing an analytics post for `submitted` events, but I am not seeing an analytics post for `clickEvents` in this component.

Reply:
@colganmeanor @jakemer 
Decided to use a fixed list of events we wanna track with Analytics instead of trying to get them dynamically.

https://github.com/justifi-tech/web-component-library/pull/529/files#diff-b38ac37a9b31c34f4641f64952f06108d22b0e903b20f59ee82b921dff2ec425